help/destinations/catalog/email-marketing/mailchimp-interest-categories.md

```diff
@@ -0,0 +1,239 @@
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+| Email | Contact email Address | Mandatory |
+
+{style="table-layout:auto"}
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> For each selected segment in Platform, the corresponding [!DNL Mailchimp Interest Categories] segment status gets updated with its segment status from Platform.</li></ul>|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+>
+>To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]**, search for [!DNL Mailchimp Interest Categories]. Alternatively you can locate it under the **[!UICONTROL Email marketing]** category.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields below and select **[!UICONTROL Connect to destination]**.
+
+| [!DNL Mailchimp] destination | [!DNL Mailchimp] |
+| --- | --- |
+| **[!UICONTROL Username]** | Your [!DNL Salesforce Marketing Cloud] domain prefix. <br>For example if your domain is <br> *`mcq4jrssqdlyc4lph19nnqgzzs84`.login.exacttarget.com*, <br> you need to provide `mcq4jrssqdlyc4lph19nnqgzzs84` as the value. |
+| **[!UICONTROL Password]** | Your [!DNL Mailchimp] **API Key**. <br> This is the API Key you had noted down in the [Gather [!DNL Mailchimp] credentials](#gather-credentials) section is of the form *key-dc*. <br>You can provide either the *key* portion or the entire form.<br> For example if your API Key is <br>*`0123456789abcdef0123456789abcde-us14`*,<br> you could provide either *`0123456789abcdef0123456789abcde`* or *`0123456789abcdef0123456789abcde-us14`* as the value. |
```

Comment:
> This is the API Key you had noted down in the [Gather [!DNL Mailchimp] credentials](#gather-credentials) section is of the form *key-dc*.

Please revise the sentence above as it does not make grammatical sense.

Also, comma after **for example**